src/oc/code.cpp

```diff
@@ -293,6 +314,10 @@ int hoc_stack_type() {
     return get_legacy_int_type(get_stack_entry_variant(0));
 }
 
+bool hoc_stack_type_is_ndim(int i) {
```

Comment:
It looks like `i` is always zero. Maybe better to drop the argument? At least if we have any ambition to move further towards `std::stack` from the current implementation.